### PR TITLE
make (channel/query) close button go to first available channel

### PR DESCRIPTION
**Pull request by ItsOnlyBinary**
#9

### CODE REVIEW

File: src/libs/state.js [hunk 1]
```diff
@@ -684,7 +684,9 @@ const state = new Vue({
             }
 
             if (isActiveBuffer) {
-                this.setActiveBuffer(network.id, network.serverBuffer().name);
+                let target = (network.buffers.length > 1) ?
+                    network.buffers[1].name : network.serverBuffer().name;
```
Comment (prawnsalad) [inline src/libs/state.js]: This ternary should use the existing format used throughout the code:
~~~
let target = expr ?
    val1 :
    val2;
~~~

**Comment by ItsOnlyBinary** [comment]
While this works and is much better than the current situation, it appears old client behaves slightly differently.
on the old client it appears to switch to the most recently visible channel.

**Comment by CoryChaplin** [comment]
@ItsOnlyBinary I think this should go to previous open window (with a store of a history we roll down) like it was un kiwi v1.

**Comment by ItsOnlyBinary** [comment]
@CoryChaplin see https://github.com/kiwiirc/kiwiirc/pull/216 ;)

**Pull request closed without merging**